packages/web/src/features/languages/ManageLanguageView.tsx

```diff
@@ -108,6 +110,8 @@ export default function ManageLanguageView() {
 
   const { data: language } = useLicenseQuery(params.code);
   const { data: members } = useLicenseMembersQuery(params.code);
```

Comment:
@arrocke What is the `useLicense` all about? It's confusing me. Is `useLoaderData` an appropriate way to load my list of font options?

Reply:
Lol. That function should be named `useLanguageQuery`. At my job, we have licenses as our top level org unit rather than languages. Probably not the last time I'll interchange them.

`useLoaderData` is the pattern for react router for loading data before the navigation completes. That should work great in this case. The new view won't render until all of the data has been fetched